pages/docs/reference/using-gradle.md

```diff
@@ -58,6 +58,27 @@ apply plugin: 'org.jetbrains.kotlin.<...>'
 </div>
 </div>
 
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
+
+
+```kotlin
```

Comment:
As [the `plugins { ... }` DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) and the Gradle plugin portal become more popular, we may want to restructure the plugin usage instructions to recommend using just this in the first place:
```
plugins {
    kotlin("jvm") version "..." // it's `id 'org.jetbrains.kotlin.jvm' version '...'`  in Groovy
}
```
We should still mention this way to apply the plugin as a fallback option.

Reply:
+